### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by crazycatman**
* Added profile page
  * Display user first and last name
  * Display joined date
  * User can toggle private mode
  * Profile picture
    * Changes based on profile visibility

### CODE REVIEW

File: src/pages/Users/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import Grid from '@mui/material/Unstable_Grid2';
+import { isErrorResponse } from 'models/ErrorResponse';
+import { useRecoilState, useRecoilValue, useSetRecoilState } from 'recoil';
+import { useTranslationWrapper } from 'services/Translation';
+import { apiState } from 'state/apiState';
+import { errorState } from 'state/errorState';
+import { whoamiState } from 'state/userState';
+
+export const Profile = () => {
+  // const user = useRecoilValue(whoamiState);
```
Comment (peseb) [inline src/pages/Users/Profile.tsx:11]:
```suggestion
```

File: src/pages/Users/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <Box marginTop={5} marginLeft={5}>
+      <Typography variant="h2">{t('profile.profile')}</Typography>
+      <Typography variant="h3"> U did boo-boo </Typography>
+    </Box>
+  ) : (
+    <Box>
+      <Box marginTop={5} marginLeft={5}>
+        <Typography variant="h2">{t('profile.profile')}</Typography>
+        <Typography variant="h5">
+          Name: {user.name} {user.last_name}
```
Comment (peseb) [inline src/pages/Users/Profile.tsx:47]: Putte Name og Joined i translation? :D

**Review by peseb — CHANGES_REQUESTED** [review]

File: src/pages/Users/Profile.tsx (hunk 1 above)
Comment (crazycatman) [reply src/pages/Users/Profile.tsx]: Fixed

File: src/pages/Users/Profile.tsx (hunk 2 above)
Comment (crazycatman) [reply src/pages/Users/Profile.tsx]: Fixed

**Review by sixcare — CHANGES_REQUESTED** [review]

**Pull request merged**